backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientMutationResolver.java

```diff
@@ -35,9 +42,15 @@ public PatientMutationResolver(PersonService ps, UploadService us) {
         _us = us;
     }
 
-    public String uploadPatients(Part part) throws Exception {
-        InputStream people = part.getInputStream();
-        return _us.processPersonCSV(people);
+    public String uploadPatients(Part part) {
+        try (InputStream people = part.getInputStream()) {
+            return _us.processPersonCSV(people);
+        } catch (IllegalGraphqlArgumentException e) {
+            throw e;
```

Comment:
Did you want to `LOG.error()` here?

Reply:
These are all validation errors, so I just want to push them forward. Previously, all exceptions were just percolating up from this method, and this change keeps that behavior for `IllegalGraphqlArgumentException`s but handles `IOException`s separately.